testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py

```diff
@@ -3,7 +3,7 @@
 
 from CBLClient.Database import Database
 from CBLClient.DatabaseConfiguration import DatabaseConfiguration
-
+from CBLClient.Query import Query
```

Comment:
seems you moved the test case to Query section, therefore this import is no longer needed?

Reply:
done. Sorry forgot to run flake8. I ran and fixed flake8 issues.